jwala-webapp/src/main/webapp/resources/js/error-alert.js

```diff
@@ -45,15 +45,7 @@ $.extend({ errorAlert: function (message, dlgTitle, modal, content) {
             }
 
             var detailsHtml = "";
-            if (content && content !== "") {
-                let details = content;
-                if (!Array.isArray(content)) {
-                    details = new Array(content.length);
-                    for (let key in content) {
-                        details.push(content[key] +"<br/>");
-                    }
-                }
-                content = details;
+            if (content && content !== "" && Array.isArray(content)) {
```

Comment:
Please remove unnecessary content validation, this will do 

    if (Array.isArray(content)) {
        ...
    }